src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Compilation.cs

```diff
@@ -37,6 +37,8 @@ public abstract class Compilation : ICompilation
         private readonly AssemblyGetExecutingAssemblyMethodThunkCache _assemblyGetExecutingAssemblyMethodThunks;
         private readonly MethodBaseGetCurrentMethodThunkCache _methodBaseGetCurrentMethodThunks;
 
+        private readonly FeatureSwitchManager mgr;
```

Comment:
This file is still not done yet - I just want it to run the mark&sweep on all IL in all tests to weed out bugs.